contrib/registry/eureka/client.go

```diff
@@ -129,18 +129,26 @@ type Client struct {
 	lock              sync.Mutex
 }
 
-func NewClient(urls []string, opts ...ClientOption) *Client {
-	tr := &http.Transport{
-		MaxIdleConns: maxIdleConns,
-	}
+var clientTransport = &http.Transport{
+	DialContext: (&net.Dialer{
+		Timeout:   5 * time.Second,
+		KeepAlive: 30 * time.Second,
+	}).DialContext,
+	IdleConnTimeout:       30 * time.Second,
```

Comment:
It should be set to 90 seconds, which needs to be greater than the connection KeepAlive time.